packages/desktop-client/src/components/sidebar/Accounts.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState, useMemo } from 'react';
+
+import Add from '../../icons/v1/Add';
+import View from '../common/View';
+
+import { Account } from './Account';
+import { SecondaryItem } from './SecondaryItem';
+
+const fontWeight = 600;
+
+export function Accounts({
+  accounts,
+  failedAccounts,
+  updatedAccounts,
+  getAccountPath,
+  allAccountsPath,
+  budgetedAccountPath,
+  offBudgetAccountPath,
+  getBalanceQuery,
+  getAllAccountBalance,
+  getOnBudgetBalance,
+  getOffBudgetBalance,
+  showClosedAccounts,
+  onAddAccount,
+  onToggleClosedAccounts,
+  onReorder,
+}) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we're missing the prop type here.
```suggestion
}: AccountsProps) {
```

Reply:
Added the props :)